src/Spreadsheet.js

```diff
@@ -224,7 +227,13 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
           <tr>
             <th />
             {range(columns).map(columnNumber => (
-              <ColumnIndicator key={columnNumber} column={columnNumber} />
+              <ColumnIndicator
+                hasLabel={Boolean(columnLabels)}
+                key={columnNumber}
+                column={
```

Comment:
Please pass label prop here `columnLabels[columnNumber]` which might be `undefined`.